runner/gateway.openapi.yaml

```diff
@@ -2,7 +2,7 @@ openapi: 3.1.0
 info:
   title: Livepeer AI Runner
   description: An application to run AI pipelines
-  version: ''
+  version: v0.7.0
```

Comment:
Not sure why these versions have been set and whether that's ok

Reply:
@mjh1 thanks for the pull request. Yea the version is not really used anymore now we use speakeasy. The version was accidentally removed during the LLM merge.